docs/framework/network-programming/automatic-proxy-detection.md

```diff
@@ -16,32 +16,30 @@ helpviewer_keywords:
   - "WPAD (Web Proxy Auto-Discovery)"
 ms.assetid: fcd9c3bd-93de-4c92-8ff3-837327ad18de
 ---
-# Automatic Proxy Detection
-Automatic proxy detection is a process by which a Web proxy server is identified by the system and used to send requests on behalf of the client. This feature is also known as Web Proxy Auto-Discovery (WPAD). When automatic proxy detection is enabled, the system attempts to locate a proxy configuration script that is responsible for returning the set of proxies that can be used for the request. If the proxy configuration script is found, the script is downloaded, compiled, and run on the local computer when proxy information, the request stream, or the response is obtained for a request that uses a <xref:System.Net.WebProxy> instance.  
-  
- Automatic proxy detection is performed by the <xref:System.Net.WebProxy> class and can employ request-level settings, settings in configuration files, and settings specified using the Internet Explorer **Local Area Network (LAN)** dialog box.  
+# Automatic proxy detection
+
+Automatic proxy detection is the process of identifying a web proxy server and sending requests on behalf of the client. This feature is also known as Web Proxy Auto-Discovery (WPAD). When automatic proxy detection is enabled, the system attempts to locate a proxy configuration script that's responsible for returning the set of proxies that can be used for the request. If the proxy configuration script is found, the script is downloaded, compiled, and run on the local computer when proxy information, the request stream, or the response is obtained for a request that uses a <xref:System.Net.WebProxy> instance.  
   
-> [!NOTE]
-> You can display the Internet Explorer **Local Area Network (LAN) Settings** dialog box by selecting **Tools** from the Internet Explorer main menu and then selecting **Internet Options**. Next, select the **Connections** tab, and click **LAN Settings**.  
+ Automatic proxy detection is performed by the <xref:System.Net.WebProxy> class and can employ request-level settings, settings in configuration files, and computer proxy settings.
   
  When automatic proxy detection is enabled, the <xref:System.Net.WebProxy> class attempts to locate the proxy configuration script as follows:  
   
-1. The WinINet `InternetQueryOption` function is used to locate the proxy configuration script most recently detected by Internet Explorer.  
+1. The WinINet `InternetQueryOption` function is used to locate the proxy configuration script most recently detected by the browser.
```

Comment:
❗ ❗ ❗ 
I wouldn't change anything related to Internet Explorer here without confirmation from the product group. **This technology is old and is probably coded for IE specifically**. When a statement like one changed here mentions that it checks IE directly, it's probably detecting IE directly and not trying to figure out the specific browser the user has installed as default.

I would revert this file.